bin/test

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+stack build
+
+stack exec reflow -- -w 78 --ignore-headers fixtures/input.txt > /tmp/output.txt
+
```

Comment:
My _main_ concern here is testing the parser/wrapper components. I'd rather not waste time running it twice. That being said, maybe this makes more sense as the way I should run the tests in general?